### PR TITLE
Update TRACONBoundaries.geojson

### PR DESCRIPTION
**Pull request by ikv20009**

### CODE REVIEW

**Comment by maiuswong** [comment]
_TWR prefixes won't work in the way intended here unless you are planning to log-in with UITT_TWR_APP.

File: TRACONBoundaries.geojson [hunk 1]
```diff
@@ -490,6 +490,12 @@
...
...07.71111,58.18389],[107.63167,58.16222],[107.55694,58.13639],[107.48806,58.10639],[107.42556,58.07250],[107.37000,58.03556],[107.32222,57.99583],[107.28278,57.95361],[107.25194,57.90944],[107.23000,57.86361],[107.21722,57.81694],[107.21361,57.76972],[107.21944,57.72278],[107.23417,57.67639],[107.25806,57.63083],[107.29056,57.58694],[107.33139,57.54528],[107.38000,57.50583],[107.43583,57.46944],[107.49833,57.43639],[107.56694,57.40694],[107.64083,57.38167],[107.71889,57.36056],[107.80056,57.34389],[107.88528,57.33167],[107.97139,57.32444],[108.05889,57.32194],[108.14611,57.32444],[108.23222,57.33167],[108.31694,57.34389],[108.39861,57.36056],[108.47667,57.38167],[108.55056,57.40694],[108.61889,57.43639],[108.68167,57.46944],[108.73750,57.50583],[108.78611,57.54528],[108.82694,57.58694],[108.85944,57.63083],[108.88306,57.67639],[108.89806,57.72278],[108.90389,57.76972],[108.90028,57.81694],[108.88750,57.86361],[108.86556,57.90944],[108.83472,57.95361],[108.79528,57.99583],[108.74750,58.03556],[108.69194,58.07250],[108.62944,58.10639],[108.56056,58.13639],[108.48583,58.16222],[108.40639,58.18389],[108.32306,58.20111],[108.23667,58.21333],[108.14833,58.22083],[108.05889,58.22333]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:496]: Note what @maiuswong said in the main conversation thread. This won't work, and isn't intended to work this way. This project is for TRACON/Approach sectors only, not TWR. Adding in a _TWR prefix will not work.

File: TRACONBoundaries.geojson [hunk 2]
```diff
@@ -490,6 +490,12 @@
...
...05.39028,57.26917],[105.31278,57.24750],[105.24000,57.22167],[105.17278,57.19167],[105.11167,57.15806],[105.05778,57.12111],[105.01111,57.08111],[104.97250,57.03889],[104.94250,56.99472],[104.92111,56.94889],[104.90861,56.90222],[104.90500,56.85528],[104.91056,56.80806],[104.92500,56.76167],[104.94833,56.71611],[104.98000,56.67250],[105.01972,56.63056],[105.06722,56.59139],[105.12167,56.55500],[105.18250,56.52167],[105.24944,56.49250],[105.32139,56.46694],[105.39778,56.44583],[105.47750,56.42917],[105.56000,56.41694],[105.64417,56.40972],[105.72917,56.40722],[105.81444,56.40972],[105.89861,56.41694],[105.98111,56.42917],[106.06083,56.44583],[106.13722,56.46694],[106.20917,56.49250],[106.27611,56.52167],[106.33694,56.55500],[106.39139,56.59139],[106.43889,56.63056],[106.47861,56.67250],[106.51028,56.71611],[106.53361,56.76167],[106.54806,56.80806],[106.55361,56.85528],[106.55000,56.90222],[106.53750,56.94889],[106.51611,56.99472],[106.48611,57.03889],[106.44750,57.08111],[106.40083,57.12111],[106.34694,57.15806],[106.28583,57.19167],[106.21861,57.22167],[106.14583,57.24750],[106.06833,57.26917],[105.98694,57.28639],[105.90278,57.29861],[105.81639,57.30611],[105.72917,57.30861]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:497]: Same issue here. This project isn't for Tower positions.

File: TRACONBoundaries.geojson [hunk 3]
```diff
@@ -502,6 +508,21 @@
...
+	  {"type":"Feature","properties":{"id":"UNBB","prefix":["UNBB_TWR"],"name":"Barnaul Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[82.30750,54.03389],[82.22306,53.55889],[82.29556,53.34417],[82.82500,53.22500],[82.86306,53.17167],[84.27806,53.28000],[84.50167,53.50806],[84.23306,53.90806],[83.98306,54.06667],[83.40306,54.12167],[82.78306,54.02667]]]]}},
+    {"type":"Feature","properties":{"id":"UNBG","prefix":["UNBG_TWR"],"name":"Gorno-Altaysk Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[84.99417,52.38306],[85.36306,51.73500],[86.05000,51.78306],[86.36778,51.92417],[86.40306,52.27139],[86.05917,52.54444],[85.38306,52.46667]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:513]: Same issue here. This project isn't for Tower positions.

File: TRACONBoundaries.geojson [hunk 4]
```diff
@@ -502,6 +508,21 @@
...
+    {"type":"Feature","properties":{"id":"UNOO","prefix":["UNOO_R"],"name":"Omsk Radar"},"geometry":{"type":"MultiPolygon","coordinates":[[[[73.15806,55.39000],[72.77139,55.26583],[72.53500,54.94167],[72.70806,54.65806],[73.26667,54.52500],[73.86667,54.65000],[74.09750,54.99833],[73.94167,55.27500],[73.52000,55.35667]]]]}},
+    {"type":"Feature","properties":{"id":"UNTT","prefix":["UNTT_TWR"],"name":"Tomsk Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[83.88167,57.10167],[83.20000,56.18306],[83.61667,56.10000],[83.80000,56.01667],[84.25000,56.00000],[84.66667,56.05000],[85.06667,55.80000],[86.00000,56.8306],[86.00000,57.00000],[84.97306,57.15167],[84.73667,57.13167]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:520]: Same issue here. This project isn't for Tower positions.

File: TRACONBoundaries.geojson [hunk 5]
```diff
@@ -502,6 +508,21 @@
...
+    {"type":"Feature","properties":{"id":"UNWW","prefix":["UNWW"],"name":"Novokuznetsk Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[87.50000,54.53306],[87.00000,54.51306],[85.75000,54.45000],[85.48306,54.35000],[85.38306,54.23306],[84.75000,53.98306],[85.40000,53.43306],[85.61667,53.08306],[86.46667,52.85000],[86.91806,52.43500],[87.28306,52.81667],[87.30000,53.20000],[87.56667,53.53306],[88.11667,53.70000],[87.71667,54.00000],[87.50000,54.53306]]]]}},
+    {"type":"Feature","properties":{"id":"UNKY","prefix":["UNKY_TWR"],"name":"Kyzyl Tower"},"geometry":{"type":"MultiPolygon","coordinates":[[[[93.90500,52.15000],[94.05000,51.34667],[94.79167,51.32500],[95.18306,51.56667],[95.00000,52.00000],[93.90500,52.15000]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:522]: Same issue here. This project isn't for Tower positions.

File: TRACONBoundaries.geojson [hunk 6]
```diff
@@ -502,6 +508,21 @@
...
...1111,66.24806],[87.69556,66.24056],[87.58278,66.22806],[87.47389,66.21083],[87.37000,66.18917],[87.27278,66.16306],[87.18278,66.13306],[87.10167,66.09917],[87.02944,66.06222],[86.96778,66.02222],[86.91667,65.97972],[86.87694,65.93556],[86.84889,65.88972],[86.83306,65.84306],[86.82889,65.79583],[86.83694,65.74889],[86.85694,65.70250],[86.88833,65.65694],[86.93111,65.61333],[86.98444,65.57167],[87.04778,65.53250],[87.12056,65.49611],[87.20194,65.46333],[87.29111,65.43389],[87.38667,65.40861],[87.48806,65.38750],[87.59389,65.37083],[87.70333,65.35889],[87.81528,65.35167],[87.92806,65.34917],[88.04111,65.35167],[88.15306,65.35889],[88.26250,65.37083],[88.36833,65.38750],[88.46972,65.40861],[88.56528,65.43389],[88.65444,65.46333],[88.73583,65.49611],[88.80861,65.53250],[88.87194,65.57167],[88.92528,65.61333],[88.96806,65.65694],[88.99944,65.70250],[89.01944,65.74889],[89.02750,65.79583],[89.02333,65.84306],[89.00750,65.88972],[88.97944,65.93556],[88.93972,65.97972],[88.88861,66.02222],[88.82694,66.06222],[88.75472,66.09917],[88.67361,66.13306],[88.58361,66.16306],[88.48639,66.18917],[88.38250,66.21083],[88.27361,66.22806],[88.16083,66.24056],[88.04528,66.24806],[87.92806,66.25056]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:525]: Same issue here. This project isn't for Tower positions.

**Review by kmoberg — CHANGES_REQUESTED** [review]
See the inline comments, but this project isn't for tower positions. Please remove them, and tag me once it's done, and I'll review the rest of the submission.

**Pull request closed without merging**